hmrc-mongo-metrix-play-26/src/test/scala/uk/gov/hmrc/mongo/metrix/MongoRepositorySpec.scala

```diff
@@ -14,40 +14,23 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.mongo.metrix.impl
+package uk.gov.hmrc.mongo.metrix
 
-import com.mongodb.BasicDBObject
-import org.mongodb.scala.{MongoClient, MongoDatabase}
+import org.mongodb.scala.model.IndexModel
+import org.scalatest.LoneElement
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{BeforeAndAfterEach, LoneElement}
-import uk.gov.hmrc.mongo.MongoComponent
-import uk.gov.hmrc.mongo.metrix.UnitSpec
-import uk.gov.hmrc.mongo.metrix.PersistedMetric
-import uk.gov.hmrc.mongo.test.MongoSupport
+import uk.gov.hmrc.mongo.test.{DefaultMongoCollectionSupport, MongoSupport}
 
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.DurationInt
 
-class MongoMetricRepositorySpec
+class MongoRepositorySpec
     extends UnitSpec
     with MongoSupport
```

Comment:
`MongoSupport` here is redundant, being mixed in by `DefaultMongoCollectionSupport` now?